websocketpp/transport/asio/connection.hpp

```diff
@@ -316,7 +316,7 @@ class connection : public config::socket_type::socket_con_type {
      */
     timer_ptr set_timer(long duration, timer_handler callback) {
         timer_ptr new_timer = lib::make_shared<lib::asio::steady_timer>(
-            lib::ref(*m_io_service),
+            *m_io_service,  // airtime - BXB-2993
```

Comment:
What causes the need for this? It might be nice to have description of the change (at least the first time it occurs).

Reply:
@artasano (This is an Art change)

Reply:
asio::basic_waitable_timer constructor changed and it takes intermediate object, therefore ref wrapper is no longer implicitly convertible to the required type.

Reply:
Should we have a comment to that effect? Or maybe a note indicating that we should revisit this change when/if we update websocketpp?

Just looking for a way to send a message to Future Addison indicating that this isn't an important functional change that we need to keep around.